src/components/UriViewer/UriViewer.js

```diff
@@ -148,7 +198,7 @@ export const UriViewer = _.flow(
         div({ style: { fontSize: 10 } }, ['* Estimated. Download cost may be higher in China or Australia.'])
       ])],
       () => h(Fragment, [
-        isGsUri(uri) ? 'Loading metadata...' : 'Resolving DRS file...',
+        isGsUri(uri) || isAzureUri(uri) ? 'Loading metadata...' : 'Resolving DRS file...',
```

Comment:
I think you can remove the `isAzureUri` here since we are in the GCP-only Modal

Reply:
so not quite sure, but if I remove it, then when the modal is loading for Azure case, it actually prints "Resolving DRS file....". Could be something that needs to change in the way this component it being used in both of the modals but I am not sure how to go about that

Reply:
had an error with how I was determining which modal to display, fixed it and now this is working as expected. Big thanks to @nawatts for pointing out the error!